pyethapp/jsonrpc.py

```diff
@@ -839,10 +840,10 @@ class Filter(object):
     :ivar new_logs: same as :attr:`logs`, but is reset at every access
     """
 
-    def __init__(self, chain, blocks=[], addresses=None, topics=None,
+    def __init__(self, chain, blocks, addresses=None, topics=None,
```

Comment:
My mistake. That should be `blocks=None`, and then `blocks = blocks or []` below that.